docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -63,7 +68,7 @@ Github tags are used to track the releases of EdgeX Foundry. During development
 
 *Tied to Code Release?* Yes
 
-Snaps are released for every named release of EdgeX Foundry. During development the community stages snaps to the `latest/edge` channel of the [Snapcraft Store](https://snapcraft.io/search?q=edgexfoundry). At the time of code freeze we will promote the snaps from `latest/edge` to the `latest/beta` amd `latest/candidate` channels as beta release. This beta release is to trigger validation of the release candidate. At the time of release we will promote the snaps from `latest/beta` to `latest/stable`. In addition we also create a named release track for the release. (ie: `geneva/stable`).
+The building of snaps was removed from community scope in September 2020 but are still available on the [snapcraft store](https://snapcraft.io/edgexfoundry).
```

Comment:
@tonyespy Can we get a high-level diagram here of how the Canonical Snap publishing process works? Maybe also release cadence. Like during code freeze, snap are published to beta, then are promoted to candidate channels, etc. Just so it's documented somewhere.

Reply:
Sure, I'll work with @siggiskulason on this.

Reply:
Would still like the release cadence documented.

Reply:
@ernestojeda are you asking for the snap release cadence details to be incorporated into this PR or as a follow-up to this PR? I'm fine with either, but it wasn't clear to me from your comments.

Reply:
@tonyespy Correct. Just so we can point people to this ADR if questions arise to when certain snaps would be available on the snap store.

Reply:
@tonyespy , @siggiskulason, if you'd like me to include your release cadence for snaps into my PR please let me know so you don't have to submit a new PR w/ TSC approval.

Reply:
Here's a draft description of our snap release process for EdgeX:

Canonical publishes daily arm64 and amd64 releases of the following snaps to `latest/edge` in the Snap Store.  These builds take place on the Canonical Launchpad platform and  use the latest code from the master branch of each EdgeX repository, versioned using the latest git tag.

- edgexfoundry
- edgex-app-service-configurable
- edgex-device-camera
- edgex-device-rest
- edgex-device-modbus
- edgex-device-mqtt
- edgex-device-grove
- edgex-cli (_work-in-progress_)

**Note** - this list may expand over time.

At code freeze the edgexfoundry snap revision in the edge channel is promoted to `latest/beta` and `$TRACK/beta`. Publishing to beta will trigger the Canonical checkbox automated tests, which include tests on a variety of hardware hosted by Canonical.

When the project tags a release of any of the snaps listed above, the resulting snap revision is first promoted from the edge channel to `latest/candidate` and `$TRACK/candidate`. Canonical tests this revision, and if all looks good, releases to `latest/stable` and `$TRACK/stable`.

Canonical may also publish updates to the EdgeX snaps after release to address high/critical bugs and CVEs (common vulnerabilities and exposures).

**Note** - in the above descriptions, `$TRACK` corresponds to the named release tracks (e.g. fuji, geneva, hanoi, ...) which are created for every major/minor release of EdgeX Foundry.